x/gamm/pool-models/balancer/pool_params.go

```diff
@@ -0,0 +1,78 @@
+package balancer
```

Comment:
Let's rename stableswap params to `pool_params.go` as well for consistency

Reply:
Can we do that in a separate PR?

Reply:
agreed for doing it in a separate pr

Reply:
Can we create an issue for this here so that we make sure we come back?

Reply:
https://github.com/osmosis-labs/osmosis/issues/1887